answers/Maker.md

```diff
@@ -0,0 +1,179 @@
+### Where does the governance happen?
+
+
+The main goal of the MakerDAO is keeping the value of DAI, a collateral-backed cryptocurrency, stable relative to the US Dollar (i.e. a "soft peg" stablecoin). DAI is issued and managed through a system of smart contracts running on the Ethereum blockchain. MakerDAO governance is primarily concerned with the determining the risk parameters that are used to manage the portfolio of assets backing DAI (currently just ETH, but soon others with the introduction of multi-collateral DAI) 
+
+Central to Maker's governance is the MKR token ("probably the 1st" ERC20 token on Ethereum according to Maker's [founder](https://unchainedpodcast.com/rune-christensen-of-makerdao-part-1-how-to-keep-a-crypto-collateralized-stablecoin-afloat/)). MKR is Maker's "governance" token. MKR holders vote on proposed changes to the system via "voting contracts" (smart contracts running on the Ethereum blockchain). 1 MKR = 1 vote, and there are two types of votes: "executive votes" and "governance votes".
+
+MKR holders are incentivized to vote on proposals that preserve the stability of the system so that a wide range of actors can use the system and generate stability fees, which must be paid in MKR. When a user pays the stability fee in MKR to unlock their collateral, the MKR is burned, decreasing the total supply.
+
+#### Governance votes
+
+Governance votes can be used to vote on one or multiple issues at once. They do not automatically trigger updates to the Maker system. Instead, these must be implemented manually by the Maker team. Nor are they binding resolutions. Rather, they are used to poll community sentiment towards larger, more substantial changes to the system. This can include making changes to the structure or governance processes of the Maker Foundation, including adding new Oracles, adding a new risk team (people that create and apply risk models), or adopting a new voting process. Votes can be time-limited. If the vote is time-limited, votes are tallied at the end of the voting period and a simple majority (<50%) determines the outcome.
```

Comment:
I would like to see some information on how many of these governance votes have occurred and what the outcomes are, plus a link to a more definitive source for this.

Reply:
It doesn't appear there's a website that hosts historical vote data, but have linked to a GitHub repo where there's a list of "Major Events and Milestones". Each vote appears to have an official Maker blog post with some details. Have also posted to a Medium piece that has some python code which could be useful for scraping vote data from the blockchain--though haven't looked too deeply into it.